src/index.html

```diff
@@ -17,61 +17,65 @@
     <div class="container text-center">
       <div class="row row-cols-2 row-cols-lg-5 g-2 g-lg-3">
         <div class="col">
-          <div class="p-3" type="button" class="collapse-btn" onclick="toggleCollapse('collapseContent1')">Weather</div>
-           <div class="collapse-content" id="collapseContent1" style="display: none">
-           <p>This is the collapsible content that will be shown or hidden.</p>
-           </div>
+          <div class="p-3 d-grid gap-2">
+            <button class="btn btn-primary btn-lg" type="button" data-bs-toggle="collapse" data-bs-target="#collapseContent1" aria-expanded="false" aria-controls="collapseContent1">
```

Comment:
The code I've added to the button that makes the collapse work is `data-bs-toggle="collapse" data-bs-target="#collapseContent1"`

`data-bs-toggle` tells us that that this button controls a `collapse`.
`data-bs-target` tells us which element to collapse.
`#collapseContent1` refers to an element with the id `collapseContent1`.

Reply:
`#` means id.
`.` means class.
As you may have learned already with css :)